### PR TITLE
Add chunyilyu

### PR DESCRIPTION
**Pull request by ChunyiLyu**
Adding myself as a member and approvers for eventing rabbitmq.

cc @xtreme-sameer-vohra

### CODE REVIEW

File: knative-sandbox-OWNERS_ALIASES [hunk 1]
```diff
@@ -99,6 +99,7 @@ aliases:
   - matzew
   eventing-rabbitmq-approvers:
   - benmoss
+  - chunyilyu
```
Comment (xtreme-sameer-vohra) [inline knative-sandbox-OWNERS_ALIASES:102]: nit: This file is auto-generated from the knative-sandbox.yaml file

**Comment by xtreme-sameer-vohra** [comment]
LGTM except for small comment above regarding auto-gen file
cc @vaikas @csantanapr for SC 🪄

File: knative-sandbox-OWNERS_ALIASES (hunk 1 above)
Comment (ChunyiLyu) [reply knative-sandbox-OWNERS_ALIASES:102]: Thanks! Updated

**Comment by rhuss** [comment]
@ChunyiLyu Could you please accept the GitHub invitation to knative-sandbox members ? Otherwise, we have difficulties updating the OWNER files in the repositories.

**Pull request merged**